content/docs/getting-started/github-enterprise/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Configuring GitHub Enterprise
+publishedDate: '2022-07-28T21:00:00.0Z'
+description: How to connect the Roadie catalog to GitHub Enterprise.
+---
+
+## Introduction
+
+This tutorial will guide you through the steps required to connect Roadie to your GitHub org in GitHub Enterprise.
+
+## Prerequisites
+
+Please let the Roadie support team know about your intention to configure GitHub enterprise, because we need to do something manually on your behalf to enable it.
```

Comment:
```suggestion
Please let the Roadie support team know about your intention to configure GitHub enterprise, because we need to do some manual steps to enable it for you.
```